media/src/main/java/edu/artic/media/audio/AudioPlayerService.kt

```diff
@@ -185,6 +186,20 @@ class AudioPlayerService : DaggerService(), PlayerService {
     val disposeBag = DisposeBag()
     internal var currentBitmap: Bitmap? = null
 
+    /**
+     * Set this to true to tell the system we are currently in a call-like environment.
+     *
+     * On Android Pie devices, that would let the system load a reasonable
+     * volume-settings preset. Only takes effect when a [Playing] event passes
+     * through [audioPlayBackStatus].
+     *
+     * Assumption: [AudioManager.setMode] only affects the mode for this service. When
+     * we die, the mode will revert to its previous setting.
+     *
+     * @see AudioManager.MODE_IN_COMMUNICATION
+     */
+    internal var forceHeadsetMode: Boolean = false
```

Comment:
What's controlling this? Also wouldn't false break it for the more common use case?

Reply:
This isn't being controlled by anything right now - we could make it a val, or inline it.

Reply:
As discussed offline, changes to this PR to resolve the question of `forceHeadsetMode` are out of scope. We can revisit the question later.